.travis.yml

```diff
@@ -34,6 +38,7 @@ before_script:
   #installing vpx
   - git clone https://chromium.googlesource.com/webm/libvpx > /dev/null
   - cd libvpx
+  - git checkout tags/v1.3.0
```

Comment:
Shouldn't that be `v1.4.0` ?

Or is toxcore on travis supposed to build with lowest supported version?